third_party/github.com/bazelbuild/bazel-skylib/rules/private/copy_file_private.bzl

```diff
@@ -19,13 +19,28 @@ cmd.exe (on Windows). '_copy_xfile' marks the resulting file executable,
 '_copy_file' does not.
 """
 
+def _hash_file(file):
+    # Strip the minus off of hash if it is negative.
```

Comment:
why is it a problem to have an extra hyphen appear in the generated filename?

Reply:
its not. I'm just pedantic. I'll remove the condition it 🙄